### PR TITLE
feat: use Radix for tabs implementation

### PR DESCRIPTION
**Pull request by suevalov**
# Purpose of PR

Reimplement current tabs using `radix-tabs` primitives

### CODE REVIEW

File: packages/components/tabs/src/Tabs.tsx [hunk 1]
```diff
@@ -1,40 +1,143 @@
...
-    className,
+export const Tabs = React.forwardRef<HTMLDivElement, TabsProps>(
+  (props, ref) => {
+    const {
+      defaultTab,
+      currentTab,
+      onTabChange,
+      children,
+      testId = 'cf-ui-tabs',
+      ...rest
```
Comment (massao) [inline packages/components/tabs/src/Tabs.tsx:21]: On other files we use `...otherProps`, shouldn't we use it here too??

**Review by massao — APPROVED** [review]
LGTM, just a small note

**Review by gui-santos — APPROVED** [review]
Amazing! I just +1 @massao comment

**Comment by burakukula** [comment]
Small visual issues

BEFORE:
Screenshot 2021-11-02 at 16 51 54

**Pull request merged**